spk/python3/src/requirements.txt

```diff
@@ -5,6 +5,7 @@
 #Pillow==5.3.0
 #pycurl==7.43.0.2
 pyyaml==5.3
+immutables==0.11
```

Comment:
immutables (and pyyaml) should be deactivated with # here, as these are included by compiled wheels.
as written on line 2 these are documented here for referency only

Reply:
Sure, was not sure about this :)